easybuild/easyblocks/generic/bundle.py

```diff
@@ -71,8 +72,8 @@ def __init__(self, *args, **kwargs):
         self.altroot = None
         self.altversion = None
 
-        # list of EasyConfig instances for components
-        self.comp_cfgs = []
+        # list of EasyConfig instances and their EasyBlocks for components
+        self.comp_instances = []
```

Comment:
@Thyre renaming this will have some fallout, see the `clang_aomp.py` where `self.comp_cfgs` is used...

Reply:
In that case, we should keep the old one, yeah. With the PR reverted, I revise the changes.

Reply:
Yeah... I missed that EasyBlock since it is unused. However, there is only a single usage of `self.comp_cfgs`.
When I open a new PR for this, I'll update the `clang_aomp.py` EasyBlock as well. Haven't found other usages.